include/irr/builtin/glsl/bxdf/bsdf/specular/beckmann.glsl

```diff
@@ -145,6 +145,34 @@ float irr_glsl_beckmann_smith_height_correlated_dielectric_cos_eval_wo_cache_val
         orientedEta,orientedEta2,a2);
 }
 
+float irr_glsl_beckmann_smith_height_correlated_dielectric_cos_eval_and_pdf(out float pdf, in irr_glsl_LightSample _sample, in irr_glsl_IsotropicViewSurfaceInteraction interaction, in irr_glsl_IsotropicMicrofacetCache _cache, in float eta, in float a2)
+{
+    const float ndf = irr_glsl_beckmann(a2, _cache.NdotH2);
+
+    float orientedEta, dummy;
+    const bool backside = irr_glsl_getOrientedEtas(orientedEta, dummy, _cache.VdotH, eta);
+    float orientedEta2 = orientedEta*orientedEta;
+
+    const float VdotHLdotH = _cache.VdotH*_cache.LdotH;
+    const bool transmitted = VdotHLdotH<0.0;
+
+    const float reflectance = irr_glsl_fresnel_dielectric_common(orientedEta2, abs(_cache.VdotH));
+
+    const float absNdotV = abs(interaction.NdotV);
+    const float NdotH2 = _cache.NdotH2;
+    const float NdotL2 = _sample.NdotL2;
+    const float NdotV2 = interaction.NdotV_squared;
+
+    float remainder = irr_glsl_beckmann_dielectric_cos_remainder_and_pdf_wo_clamps(
+        pdf,
+        ndf, transmitted, NdotL2, absNdotV,
+        NdotV2, _cache.VdotH, _cache.LdotH, VdotHLdotH,
+        reflectance, orientedEta,
+        a2
+    );
+
+    return remainder*pdf;
+}
```

Comment:
save us the torture of a big and complex wrapper around calling the `remainder_and_pdf` outselves at the target site

I'd much rather see a function in the top level header like this:
```glsl
float irr_glsl_bxdf_remainder_to_eval(in float remainder, in float pdf)
{
    return remainder*pdf;
}
vec3irr_glsl_bxdf_remainder_to_eval(in vec3 remainder, in float pdf)
{
    return remainder*pdf;
}
```

because its just 6 lines to maintain, as opposed to a separate `eval_and_pdf` boilerplate function for every BxDF in our library